python/lsst/daf/butler/core/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""
+Core code for butler.
+"""
+
+# Do not export the utility routines from safeFileIo and utils
+
+from .composites import *
+from .config import *
+from .datasets import *
+from .datastore import *
+from .dataUnits import *
+from .fileDescriptor import *
+from .fileTemplates import *
+from .formatter import *
+from .location import *
+from .mappingFactory import *
+from .quantum import *
+from .regions import *
+from .registry import *
+from .run import *
+from .schema import *
+from .storageClass import *
+from .units import *
```

Comment:
Are we sure we need to pull everything in?

Reply:
The big issue with the docs is working out what is public API and should be imported into `lsst.daf.butler` and which classes are entirely private API. You'll see that I didn't import the utility scripts but explicitly got those documented in a later section. This Sphinx stuff really does force us to work out public vs private.